support/ci/lint.sh

```diff
@@ -108,7 +108,7 @@ eval "$cmd" | while read file; do
       case $rf_exit in
         0|3)
           if echo "$output" | grep -q "Diff at line " >/dev/null; then
-            warn "File $file generates a diff after running rustfmt $rf_version"
+            warn "File $file generates a diff after running rustfmt"
```

Comment:
Previously we used `$rf_version` to check the version of `rustfmt`, but now we just rely on [pinning the `rust` version in `.travis.yml`](https://github.com/habitat-sh/builder/blob/master/.travis.yml#L42).